### DISCUSSION TITLE
List Tag Rendering Incorrectly With Bullet

### DISCUSSION THREAD

**Discussion by rtkaaho** [seek-oss/capsize#52] [General] [answered]
Hi there!

Working through adding capsize to the text component in our design system and am running into an issue where capsized text in an `li` tag renders on the next line of the bullet instead of inline. Is this use case not meant to be supported or is this a general issue?

#### Reproduction

You can easily reproduce it on https://seek-oss.github.io/capsize/ by changing the tag of the capsized text to an `li` tag. See the screenshot below:

![Screen Shot 2021-10-05 at 10 34 55 AM](https://user-images.githubusercontent.com/56700218/136044257-a5660d9d-ebd1-4459-9c4b-9c411715ee29.png)

**Comment by michaeltaranto** [comment, marked as answer]
Hey :wave: 
Unfortunately, there are some tradeoffs with this technique, the main one being that the text node should be self contained and not have mixed styles with its parent. Unfortunately the `li` tag injects a `marker` pseudo element who's layout we cannot control.

In our design system we have opted for custom bullets, which I acknowledge adds complexity, but if you're trimming the space on your list items, you're likely opting into managing this complexity for your consumers anyway.

Also trimming the white space on the list items means having to [manage the space with your `ul`/`ol` element](https://seek-oss.github.io/braid-design-system/playroom/#?code=N4Igxg9gJgpiBcIA8AhCAPABABwIZSgEsA7AcwF4AdEUgVwBd6YAnazAI1zAGtTmJaxKFRDEYDZrgA2AGUKkAFvWoA%2BSsUyZUGDl179Bw6mFzMoq9Zs1JaUzAGd6ATykxywYJgC2p0iXiYAAwANDj4RGQBgZgAvjFqGlbWUoQOzq7unkT22FK4TgHUgvYwyiCx8ZZJVkgAKjDo9A6EAF5u1HnMpDAWidU1Obgaji5uHmEEJKQASvJKhSAAjIHY6GxxKoBEBEgA9IPECf1Jco6YhExeVf279Y2H1bsp90lIKWmjmZjZufkLxaXrSp9B63Jr2VrtECdbq9I4DPDDdJjTx4SZkWaKegLZarQFbXb7Z5HE5Nc4wS7Al47UFE6w7J5XGpvEYZcbfPIFTBFYglMoVWk1UHNNoiaE9EACl77d6slHhKYY%2BZcpYrNblDbbPYIyVWElnC6Mqk0w1aemEIm7Wz3XZodAJG0YBIgYIgegKckwewIADaIFweDAzpAACsIOwAMr0ZgwUogAC6LoA7oQoG6vfBvQB2ABsAA44zEgA), which you will need to handle carefully to not introduce additional elements in the DOM between the list and item.

Hope that helps.
